content/millennium/r4/proprietary-codes.md

```diff
@@ -342,6 +342,50 @@ This code set is customized by clients and defines exam rooms, surgery rooms, an
       "display": "Radiology Exam Room",
     }
 
+#### Code Set 18736 Account Types
+
+This code set includes Account Types such as A/R and CASH.
+
+    {
+      "system": "https://fhir.cerner.com/<EHR source id>/codeSet/18736,
+      "code": "18736",
+      "display": "Cash",
+      "userSelected": true
+    }
+
+#### Code Set 18934 Currency Type
+
+This code set includes currency types such as USD.
+
+    {
+      "system": "https://fhir.cerner.com/<EHR source id>/codeSet/18736,
```

Comment:
The code set is wrong

Reply:
https://github.com/cerner/fhir.cerner.com/commit/62764511c7f719faa5141a6b7bcbfcc839226e9f#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R361

+1